pkg/controller/certificate-shim/helper.go

```diff
@@ -63,6 +65,52 @@ func translateAnnotations(crt *cmapi.Certificate, ingLikeAnnotations map[string]
 		crt.Spec.CommonName = commonName
 	}
 
+	if emailAddresses, found := ingLikeAnnotations[cmapi.EmailsAnnotationKey]; found {
+		crt.Spec.EmailAddresses = strings.Split(emailAddresses, ",")
+	}
+
+	subject := &cmapi.X509Subject{}
+	if organizations, found := ingLikeAnnotations[cmapi.SubjectOrganizationsAnnotationKey]; found {
+		subject.Organizations = strings.Split(organizations, ",")
+	}
+
+	if organizationalUnits, found := ingLikeAnnotations[cmapi.SubjectOrganizationalUnitsAnnotationKey]; found {
+		subject.OrganizationalUnits = strings.Split(organizationalUnits, ",")
+	}
+
+	if countries, found := ingLikeAnnotations[cmapi.SubjectCountriesAnnotationKey]; found {
+		subject.Countries = strings.Split(countries, ",")
+	}
+
+	if provinces, found := ingLikeAnnotations[cmapi.SubjectProvincesAnnotationKey]; found {
+		subject.Provinces = strings.Split(provinces, ",")
+	}
+
+	if localities, found := ingLikeAnnotations[cmapi.SubjectLocalitiesAnnotationKey]; found {
+		subject.Localities = strings.Split(localities, ",")
+	}
+
+	if postalCodes, found := ingLikeAnnotations[cmapi.SubjectPostalCodesAnnotationKey]; found {
+		subject.PostalCodes = strings.Split(postalCodes, ",")
+	}
```

Comment:
Might as well escape this too! I can't find any examples of commas in postal codes, but it's possible it could happen!